src/router/router.tsx

```diff
@@ -0,0 +1,16 @@
+import { createBrowserRouter } from 'react-router-dom';
+import LoginForm from '../components/Auth/LoginForm';
+import RegisterPage from '../components/Auth/RegisterForm';
+
+export default createBrowserRouter([
+    {
+      path: '/login',
+      element: <LoginForm/>,
```

Comment:
Here you should have pages, not components. Remember: components are just reusable blocks